packages/js-client-rest/src/qdrant-client.ts

```diff
@@ -36,7 +40,17 @@ export class QdrantClient {
     private _restUri: string;
     private _openApiClient: OpenApiClient;
 
-    constructor({url, host, apiKey, https, prefix, port = 6333, timeout = 300_000, ...args}: QdrantClientParams = {}) {
+    constructor({
+        url,
+        host,
+        apiKey,
+        https,
+        prefix,
+        port = 6333,
+        timeout = 300_000,
+        check_compatibility = true,
```

Comment:
```suggestion
        checkCompatibility = true,
```
Following the convention.